File: 251221/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.py
a, b, c = map(int, input().split())

min_num = a

if b < a:
    if b < c: min_num = b
    else: min_num = c
elif c < a:
    if b < c: min_num = b
    else: min_num = c

if min_num == a:
    print(1, end=' ')
else:
    print(0, end=' ')

if a == b == c:
    print(1)
else:
    print(0)

File: 251221/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.js
const fs = require("fs");
const [a, b, c] = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let res1 = 0;
let res2 = 0;
if (a <= b && a <= c) {
    res1 = 1;
}

if (a == b && b == c) {
    res2 = 1;
}

console.log(res1, res2)